inst/WORDLIST

```diff
@@ -1,84 +1,965 @@
+️
```

Comment:
The first line needs removed because there in a non UTF-8 character on this line that makes the file unreadable by a Windows computer.

Reply:
Good catch. An emoji in the testing cheat sheet caused the issue. I replaced it with a markdown emoji, so the non-UTF-8 character problem is fixed.